header/header.go

```diff
@@ -42,6 +43,29 @@ type ExtendedHeader struct {
 	DAH          *DataAvailabilityHeader `json:"dah"`
 }
 
+func (eh *ExtendedHeader) New() libhead.Header {
+	return new(ExtendedHeader)
+}
+
+func (eh *ExtendedHeader) ChainID() string {
+	return eh.RawHeader.ChainID
+}
+
+func (eh *ExtendedHeader) Height() int64 {
+	return eh.RawHeader.Height
+}
+
+func (eh *ExtendedHeader) Time() time.Time {
+	return eh.RawHeader.Time
+}
+
+func (eh *ExtendedHeader) Validate() error {
+	// TODO(tzdybal): rename ValidateBasic to Validate
+	return eh.ValidateBasic()
```

Comment:
Does anything prevent us from doing this already?

Reply:
No. Renamed.